### DISCUSSION TITLE
Why disable a plugin instead of removing it?

### DISCUSSION THREAD

**Discussion by zubin** [LazyVim/LazyVim#293] [Q&A] [answered]
The [❌ Disabling Plugins](https://www.lazyvim.org/configuration/plugins#-disabling-plugins) section explains _how_ to disable plugins, but it would be good to understand _why_.

**Comment by folke** [comment, marked as answer]
What do you mean? Disabling a plugin **IS** removing the plugin.

**Reply by zubin** [reply]
Perhaps I'm not understanding something, but I meant why not delete the plugin from config (ie not installed) instead of setting `enabled: false` (ie installed but not enabled).

**Reply by folke** [reply]
If you set `enabled=false` and then do `:Lazy clean` or `:Lazy sync`, the plugin will bre removed and won't be installed again.

**Reply by zubin** [reply]
Ah ok thanks, that's good to know. 

But I'm still a little perplexed: why make mention of a disabled/removed plugin at all? Is it for convenience, so that it can be re-enabled easily?

**Reply by dpetka2001** [reply]
Because it's part of the default configuration of Lazyvim and maybe not everyone would like to remove it? So the option to disable a plugin (which also removes it) is there for everyone who would like to make use of it.

**Reply by zubin** [reply]
Thanks, that makes sense. 👍
